de/news/_posts/2017-09-14-openssl-asn1-buffer-underrun-cve-2017-14033.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: news_post
+title: "CVE-2017-14033: Pufferunterlauf-Problem bei Dekodierung von OpenSSL-ASN1"
+author: "usa"
+translator: "Marvin Gülker"
+date: 2017-09-14 12:00:00 +0000
+tags: security
+lang: de
+---
+
+Es gibt ein Sicherheitsproblem mit einem Pufferunterlauf in der in
+Ruby enthaltenen OpenSSL-Bibliothek, dem die CVE-Nummer
+[CVE-2017-14033](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14033)
+zugewiesen worden ist.
+
+## Details
+
+Wenn ein böswilliger String an die Methode `decode` von
+`OpenSSL::ASN1` übergeben wird, kann ein Pufferunterlauf verursacht
+werden, der zum Absturz des Ruby-Interpreters führen kann.
+
+Alle Nutzer einer betroffenen Version sollten entweder aktualisieren
+oder umgehend einen der folgenden Workarounds anwenden.
+
+## Betroffene Versionen
+
+* 2.2er Serie: 2.2.7 und früher
+* 2.3er Serie: 2.3.4 und früher
+* 2.4er Serie: 2.4.1 und früher
+* Trunk vor Revision 56946
+
+## Workaround
+
+Die OpenSSL-Bibliothek wird auch als Gem verteilt. Wenn Sie Ruby
+selbst nicht aktualisieren können, installieren Sie den OpenSSL-Gem in
+einer Version neuer als 2.0.0. Dieser Workaround funktioniert
+allerdings nur mit der 2.4er Serie von Ruby, denn bei der 2.2er und
```

Comment:
2.3er-Serie etc.

Reply:
No. Like in "1990er Jahre", "2.3er Serie" does not have a hyphen. At least that's how I intuitively would think, I don't have a print version of the Duden to find out.

Reply:
Duden § 42 includes the following examples: 100stel-Millimeter, 61er-Bildröhre, 25er-Gruppe; and: "in den 80er-Jahren (_auch_ in den 80er Jahren)"; so the normal case is with hyphen.

Reply:
You won. But I swear I have learned it back in school without the hyphen. But these details actually change quick...